src/themes/files/dark/primitives.json

```diff
@@ -573,6 +573,10 @@
       "$value": "Inter"
     },
     "code": {
+      "$type": "string",
+      "$value": "IBM Plex Mono"
+    },
+    "code-alternate": {
```

Comment:
I don't know if it's possible, but maybe instead of creating an alternate var we could leverage the CSS functionality that allows you to list fonts in order of preference (e.g. putting "IBM Plex Mono, Menlo" as the value)

Reply:
We can actually do that.
I'll tag @FrankieBortot to check it out with him too.

Reply:
That's not really how tokens work: each token represent a clear and specific value, not a series of value.
Also, Figma doesn't allow that 🥲
I came up with "code-alternate" because 
1. it's a naming convention we already have
2. didn't want to change the original "code" token, so i hade little room for changes 
What would you suggest instead of "code-alternate"?

Reply:
Ok so it makes sense that the token should have only one value. 

My issue with the code-alternate here is the actual usefulness of the token: since that font is proprietary with a very strict license, and will hardly ever be used directly (I mean included in the project, with the license and all), the only use I see is as a fallback if the IBM one fails to be downloaded. The fallback applies only to Mac users, therefore in my opinion it's a bit too much to have it as a standalone value.

Reply:
I agree, we can remove it from the primitives.
And we on Figma can have a separate collection for special "external" tokens: this way we don't pollute tokens with unwanted value, but at the same time we can have our primiteves/semantic separated from the rest if needed

Reply:
Removed `code-alternate`